sim/hunter/raptor_strike.go

```diff
@@ -122,6 +134,11 @@ func (hunter *Hunter) getRaptorStrikeConfig(rank int) core.SpellConfig {
 			if hasMeleeSpecialist && sim.RandomFloat("Raptor Strike Reset") < 0.3 {
 				spell.CD.Reset()
 			}
+
+			if hasRaptorFury {
+				if !hunter.RaptorFuryAura.IsActive() { hunter.RaptorFuryAura.Activate(sim) }
```

Comment:
🔨 Tiny Knit: It looks like the last `}` didn't get pushed to the next line

Reply:
I think technically you don't have to check `IsActive()` here either and can just call 
```
hunter.RaptorFuryAura.Activate(sim)
hunter.RaptorFuryAura.AddStack(sim)
```

back-to-back

Reply:
Actually the way he has it is correct. The ability states that it does not refresh its duration but keeps on stacking. So this is correct. But I do agree I dont like the formatting. Make sure you're following the styling options of the project which means new line on every curly bracket

Reply:
> Actually the way he has it is correct. The ability states that it does not refresh its duration but keeps on stacking. So this is correct.

Ah good catch!

Reply:
Yeah I thought this was funny because my other PR I had the opposite problem where I wasn't refreshing when I should've been, I'll also include the formatting changes